openeo_processes_dask/process_implementations/cubes/apply.py

```diff
@@ -9,8 +11,18 @@
 def apply(
     data: RasterCube, process: Callable, context: Optional[dict] = None, **kwargs
 ) -> RasterCube:
-    parameters = {"x": data, "context": context}
```

Comment:
illuminating what context and positional_parameters are used for in a docstring could be pretty useful here, same for context in reduce

Reply:
Added comments to document this!